.github/workflows/helm.yaml

```diff
@@ -56,6 +56,26 @@ jobs:
         run: |
           git config --global user.name "$GITHUB_ACTOR"
           git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
+      
+      - name: Set up Go 1.17
+        id: go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Run helm-docs
+        run: |
+          cd ${GITHUB_WORKSPACE}
+          GOBIN=$PWD GO111MODULE=on go install github.com/norwoodj/helm-docs/cmd/helm-docs@v1.6.0
+          ./helm-docs --chart-search-root=${GITHUB_WORKSPACE}/charts
+          git diff --exit-code ${GITHUB_WORKSPACE}/charts/ingress-nginx/README.md
+          if [ $? -ne 0 ]; then
+             git add ${GITHUB_WORKSPACE}/charts/ingress-nginx/README.md
+             git commit -m "Update helm README"
+             git push --force --quiet https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git main > /dev/null 2>&1 
```

Comment:
force push is a dangerous operation. I will try to verify in my forked repo.

Reply:
I can remove it from here and we check without it